### PR TITLE
docs: alert about support being limited to Nuxt 2

### PR DESCRIPTION
**Pull request by robsonsobral**
I also added a link to @manniL 's recipe.

image

### CODE REVIEW

File: docs/content/en/index.md [hunk 1]
```diff
@@ -8,7 +8,16 @@ category: ''
 <img src="/preview.png" class="light-img" width="1280" height="640" alt=""/>
 <img src="/preview-dark.png" class="dark-img" width="1280" height="640" alt=""/>
 
-[Sentry Module](https://github.com/nuxt-community/sentry-module) for [NuxtJS](https://nuxtjs.org).
+[Sentry Module](https://github.com/nuxt-community/sentry-module) for [NuxtJS 2](https://v2.nuxt.com/).
+
+
+<alert type="warning">
+
+  If you want support for Nuxt 3, please, raise your opinion [in this discussion](https://github.com/getsentry/sentry-javascript/discussions/6929) and ensure to make clear that the Nuxt community would love to see a first-class support from Sentry 🔥.
```
Comment (rchl) [inline docs/content/en/index.md:16]: I don't think they will appreciate people who are adding "me too" comments. It should be enough to show support by adding reaction on the first post there.

File: docs/content/en/index.md [hunk 2]
```diff
@@ -8,7 +8,16 @@ category: ''
...
+
+  If you can't wait, there's a recipe [in this post](https://www.lichter.io/articles/nuxt3-sentry-recipe/) you can try.
```
Comment (rchl) [inline docs/content/en/index.md:18]:
```suggestion
  In the mean time you can try a [solution that can provide limited functionality](https://www.lichter.io/articles/nuxt3-sentry-recipe/).
```

Comment (robsonsobral) [reply docs/content/en/index.md:18]: I'm sorry for not commit your suggestion, but there's a typo. I'll submit an update shortly.

**Comment by robsonsobral** [comment]
Updated messages:

image

**Review by rchl — APPROVED** [review]

**Pull request merged**